submissions/Vitaminvp/HTML&CSSPractice/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Responsive Side Menu</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<header>
+  <nav class="nav">
+      <input type="checkbox" class="nav__input" id="nav__input">
+      <label for="nav__input">
```

Comment:
Putting divs into labels is not valid. How about using span instead of divs here?

Reply:
Thanks, for compliments. 
I'm not shure. Could you give me some link to investigate this question more diply. Cos I think in HTML5 it isn't an issue any more. And my IDE tells me that it's ok. For me it will cose an extra "display: block;" rule.